core/templates/domain/skill/skill-creation-backend-api.service.ts

```diff
@@ -30,6 +31,7 @@ export interface SkillCreationBackendDict {
   'explanation_dict': string;
   'linked_topic_ids': string[];
   'rubrics': RubricBackendDict;
+  files?: Record<string, unknown>;
```

Comment:
Why this is optional (`?`) and contains `unknown`?

Reply:
The optional ? is not needed and I have removed it now. The type unknown is taken from `image local storage service`

Reply:
You should not use `unknown` when not needed. Can you find a proper replacement?

Reply:
I have changed the type to string because the function `getFilenameToBase64MappingAsync` returns a mapping of filename (string) to base64 version of image (string).